### ISSUE TITLE
[ONNX Importer] Replace symbolic dimension sizes with 1

### ISSUE THREAD

**Issue by mciprian13** [pytorch/glow#3791]
A lot of ONNX models obtained through conversion from TensorFlow models contain symbolic tensor sizes (allowed by the ONNX format through the "dim_param" specifier). This mostly occur for the batch size of the model like the example below:
![image](https://user-images.githubusercontent.com/29785908/69080470-bc370e00-0a44-11ea-8438-1ae01f600509.png)
Glow currently allows only statically typed tensor sizes but it should be capable to import models with symbolic sizes by forcing their value to "1" (I think it is a safe approach).

The tensor description in the proto looks like this:
```
  input {
    name: "data"
    type {
      tensor_type {
        elem_type: 1
        shape {
          dim {
            dim_param: "None"
          }
          dim {
            dim_value: 3
          }
          dim {
            dim_value: 416
          }
          dim {
            dim_value: 416
          }
        }
      }
    }
  }
```

**Comment by tlepley-cadence** [comment]
When converting Tensorflow models to ONNX models (with TF2ONNX), I had noticed that the batch dimension was always replaced by 1 in the ONNX model. 

Which TF->ONNX converter did you use to generate this ONNX model with a "None" batch dimensions ?

**Comment by mciprian13** [comment]
I think it was the MMDNN tool:
https://github.com/microsoft/MMdnn

**Comment by mciprian13** [comment]
This issue was solved in #3817.
